### PR TITLE
P2: Add branding to Reader > Settings > Followed Sites

### LINKED ISSUE #15473: P2: Reader > Settings > Followed Sites > site icons contain gray tint
### Expected behavior
When navigating to the Followed Sites list, site icons that contain transparent areas should not be tinted gray.

### Actual behavior
Site icons are tinted gray. Transparent areas of the image should let the background color show through.
<a href="https://user-images.githubusercontent.com/1062444/101919992-ddf3bd80-3b90-11eb-94ca-c1500d71b46b.png"></a>

### Steps to reproduce the behavior
1. Navigate to the Reader tab > Settings (gear icon) > Followed sites. 
2. Scroll for a bit.
3. Notice sites that contain a site icon with transparencies have a gray overlay or gray tinge.

### Notes
While we're here to fix the site icon, add the new P2 branding by changing the default site icon to the P2 logo (for P2 sites only).

### PR DESCRIPTION
**Pull request by mindgraffiti**
Fixes #15473 

This PR fixes the bug found in #15473 and updates the default site icons for P2 sites.

### To test
1. Navigate to Reader tab > Settings (gear icon) > Followed sites.
2. Scroll for a bit.
3. Notice that site icons that contain transparencies are no longer tinged gray, but instead let the background color come through.

### Screenshots

| Bug | Fix in light mode | fix in dark mode |
| --- | --- | --- |
| Simulator Screen Shot - iPhone 11 Pro - 2020-12-07 at 16 15 38 | Simulator Screen Shot - iPhone 11 Pro - 2020-12-11 at 08 59 15 | Simulator Screen Shot - iPhone 11 Pro - 2020-12-11 at 08 59 43 |

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by ScoutHarris** [comment]
Question - should the P2 icon have a border? It seems odd in this context where none of the others do.

<kbd>Screen Shot 2020-12-11 at 11 23 09 AM</kbd>

**Comment by mindgraffiti** [comment]
@ScoutHarris good point. In My Sites (site picker list) the border already existed, so I updated the color, but that's not the case for this view. Updated to remove the border ✅

File: WordPress/Classes/ViewRelated/Reader/ReaderFollowedSitesViewController.swift [hunk 1]
```diff
@@ -394,10 +394,16 @@ extension ReaderFollowedSitesViewController: WPTableViewHandlerDelegate {
             return
         }
 
-        // Reset the site icon first to address: https://github.com/wordpress-mobile/WordPress-iOS/issues/8513
-        cell.imageView?.image = .siteIconPlaceholder
+        var placeholderImage: UIImage = .siteIconPlaceholder
+        if site.isP2Type {
+            placeholderImage = UIImage.gridicon(.p2, size: CGSize(width: 40, height: 40))
+            cell.imageView?.layer.borderColor = UIColor.divider.cgColor
```
Comment (ScoutHarris) [inline WordPress/Classes/ViewRelated/Reader/ReaderFollowedSitesViewController.swift]: If it doesn't have a border, it doesn't need a border color, right?

**Review by ScoutHarris — APPROVED** [review]
Just one more minor question, then :shipit: !

File: WordPress/Classes/ViewRelated/Reader/ReaderFollowedSitesViewController.swift (hunk 1 above)
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/Reader/ReaderFollowedSitesViewController.swift]: Also, if the border color can be removed, creating the `placeholderImage` can probably be reduced to something like:
`let placeholderImage = site.isP2Type ? UIImage.gridicon(.p2, size: CGSize(width: 40, height: 40)) : .siteIconPlaceholder`

Comment (mindgraffiti) [reply WordPress/Classes/ViewRelated/Reader/ReaderFollowedSitesViewController.swift]: maybe I was saving it for later, who knows 😁 
deleted ✅

**Pull request merged**